### DISCUSSION TITLE
[Feature Request]: Add NewToolbar Item "New Private Tab"

### DISCUSSION THREAD

**Discussion by devxbasit** [Floorp-Projects/Floorp#706] [Feature Request]
It would be helpful to have another item for "New Private **Tab**", currently there is no way to open a new private tab with a single click, thank you

![image](https://github.com/Floorp-Projects/Floorp/assets/24441401/85de4cf0-1528-4ff9-a5cf-3131ed903e65)

**Comment by Ryosuke-Asano** [comment]
Good Idea! I will add it
